### ISSUE TITLE
[Feature] Create Switch Login Mechanism on SiteManager Dashboard

### ISSUE THREAD

**Issue by abhinavjonnada82** [episphere/dashboard#493]
Description:
Allow the user to switch login mechanism if they logged in using email (to phone) or phone (to email).

UI Wireframe:

https://www.figma.com/file/r8jkPLdEXyXHfqrnh49rjX/Untitled?node-id=0%3A1

MVP:
- [x] Provide update email option if logged in using email
- [x] Upon a click & confirmation, log in method should switch from email to phone or vice-versa
- [x] Feature should be restricted only to Site Managers

Nice to-haves:

- [ ] Send a confirmation email/text to updated login method

**Comment by brotzmanmj** [comment]
Additional MVP requirements:
1) If the login to which you are trying to change already exists in Firebase, return an error message stating this and don't allow the change.
2) When a login change is made in Firebase, also update the corresponding CIDs in Firestore. That way, the SMDB and the MyConnect app (MyProfile page) will display the up-to-date account information .

**Comment by abhinavjonnada82** [comment]
> Additional MVP requirements:
> 
>     1. If the login to which you are trying to change already exists in Firebase, return an error message stating this and don't allow the change.
> 
>     2. When a login change is made in Firebase, also update the corresponding CIDs in Firestore. That way, the SMDB and the MyConnect app (MyProfile page) will display the up-to-date account information .

1. I believe I did include error fallback cases but let me check once more
2. I made sure recently changed CIDs do get updated in SMDB and I can do the same for My Connect app as well

**Comment by brotzmanmj** [comment]
Ready to move to stage for final testing

**Comment by cunnaneaq** [comment]
I completed testing in stage - my SSO in stage worked well.  I tested:
1. update email 2. update phone 3. phone --> email 4. email --> phone

When I went back to the PWA, I was only able to sign in with the updated information except for scenario 4.
For scenario 4, I was able to sign in using both the original email address and the new phone number.  @abhinavjonnada82 this is an outstanding issue.  Thanks!
